app/views/hackathon/digest_mailer/_hackathon.html.erb

```diff
@@ -0,0 +1,24 @@
+<article style="margin-bottom: 1.25rem;">
+  <%# Hackathon name %>
+  <div style="margin-bottom: 0.25rem;">
+    <% if hackathon.logo.attached? %>
+      <%= image_tag hackathon.logo, style: "width: 32px; vertical-align: middle; margin-right: 0.5rem;" %>
+    <% end %>
+    <h3 style="vertical-align: middle; display: inline-block; margin: 0;">
+      <%= link_to hackathon.name, hackathon.website %>
+    </h3>
+  </div>
+
+  <%# Hackathon description %>
+  <div>
+    <p style="margin: 0;">
+      <%= hackathon.starts_at.to_date.to_fs(:long_ordinal) %>
```

Comment:
`start_date` and `end_date` might be warranted under `Hackathon::Scheduled`

Reply:
Great idea

[`881cd21` (#76)](https://github.com/hackclub/hackathons-backend/pull/76/commits/881cd21c64b8d31cd9d1bac85d86e85e8de8a4f6)